client/my-sites/backup/backup-storage-space/index.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { sprintf } from '@wordpress/i18n';
+import { Card, ProgressBar } from '@automattic/components';
+import React, { FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+type Props = {
+	storageLimit: number;
+	usedStorage: number;
+};
+
+export const BackupStorageSpace: FunctionComponent< Props > = ( { storageLimit, usedStorage } ) => {
+	const translate = useTranslate();
+
+	const usedStorageFraction = usedStorage / storageLimit;
+	let color = '#2C3338';
+	if ( 0.6 <= usedStorageFraction ) {
+		color = '#DEB100';
+	}
+	if ( 0.85 <= usedStorageFraction ) {
+		color = '#E65054';
+	}
+
+	// TODO: account for MB/GB and translate once API data is available
+	const title = sprintf( '%1$sGB of %2$sGB used', usedStorage, storageLimit );
+
+	return (
+		<Card className="backup-storage-space">
+			<div className="backup-storage-space__container">
+				<div>{ translate( 'Storage space' ) }</div>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **134** times:
`translate( 'Storage Space' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/backup-storage-space-component)